core/utils.py

```diff
@@ -673,7 +672,7 @@ def get_current_time_in_millisecs() -> float:
     Returns:
         float. The time in milliseconds since the Epoch.
     """
-    return get_time_in_millisecs(datetime.datetime.utcnow())
+    return get_time_in_millisecs(datetime.datetime.now())
```

Comment:
Should we supply an explicit tz=timezone.utc argument for this? 

https://stackoverflow.com/questions/42433171/when-to-use-datetime-utcnow-or-datetime-nowtz-pytz-utc-replacetzinfo-none

https://aaronoellis.com/articles/python-datetime-utcnow-considered-harmful

Reply:
yes, agreed. It returns the same value but we should add the timezone so that we don't deal with naive datetime objects anywhere. Added, thanks!